README.md

```diff
@@ -1,2 +1,49 @@
 # vsc-test-suite
 VSC test suite
+
+# How to run the tests (old)
+
+## VUB Tier-2 cluster Hydra
+
+```shell
+module load ReFrame/3.9.1
+
+export RFM_CONFIG_FILE=$PWD/config_vsc.py
+
+reframe --verbose --run \
+    --checkpath vsc.py \
+    --setvar VSCEnvTest.valid_systems='hydra:local' \
```

Comment:
I think the setvar options are no longer necessary if the valid systems are set in the tests themselves.

Reply:
I prefer not to set them in the tests, because then you wil have to regularly update the tests, e.g. when a new cluster is in production.
doing it with `--setvar` also allows for more flexibility: you can choose to run it in specific systems instead of only in the valid ones.

Reply:
Discussed with Sam, 
1. define only the partition in valid_systems.
2. use the config file for fine tuning 
3. use --setvar for specific use cases, it conflicts with recursive